concourse/scripts/environment.sh

```diff
@@ -65,4 +65,9 @@ export BOSH_FQDN_EXTERNAL=${BOSH_FQDN_EXTERNAL}
 export VAGRANT_SSH_KEY=${PROJECT_DIR}/${DEPLOY_ENV}-vagrant-bootstrap-concourse
 export FLY_CMD=${FLY_CMD}
 export FLY_TARGET=${FLY_TARGET}
+export CONCOURSE_POSTGRES_DATABASE=${CONCOURSE_DATABASE_NAME}
+export CONCOURSE_POSTGRES_USER=${CONCOURSE_DATABASE_USER}
+export CONCOURSE_POSTGRES_PASSWORD=${CONCOURSE_DATABASE_PASS}
+export CONCOURSE_ADD_LOCAL_USER=${CONCOURSE_ATC_USER}:${CONCOURSE_ATC_PASSWORD}
+export CONCOURSE_EXTERNAL_URL=${CONCOURSE_URL}
```

Comment:
I'm not clear why these variables need to be added to the environment script given they have the same values as the similarly named variables already present here. I fell this duplication has the potential to lead to confusion.

Reply:
This is to allow us to upstream-PR the docker-file changes that consume them, in a way which upstream is hopefully likely to find acceptable.